### PR TITLE
FAD-6121 Add redirects for email links

### PR DESCRIPTION
**Pull request by jonambas**

### CODE REVIEW

File: src/config/emailRoutes.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+/**
+ * Handles redirect for existing profile route
+ * and the templates 'email-verification', 'welcome-user'
+ * /account/profile?verify={{token}}
+ * to /account/email-verification/{{token}}
+ */
+export function emailVerificationRedirect(props) {
+  const search = qs.parse(props.location.search);
+  return search.verify ? <Redirect to={`/account/email-verification/${search.verify}`}/> : <ProfilePage {...props} />;
```
Comment (bkemper) [inline src/config/emailRoutes.js]: How does this work if the user is not logged in?

File: src/config/emailRoutes.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+/**
+ * Handles redirect for template 'forgotten-password'
+ * /reset-password?token={{token}}
+ * to /reset-password/{{token}}
+ */
+function passwordReset(props) {
+  const search = qs.parse(props.location.search);
+  return <Redirect to={search.token ? `/reset-password/${search.token}` : '/'}/>;
```
Comment (bkemper) [inline src/config/emailRoutes.js]: It's a shame this doesn't work out of the box.

File: src/config/emailRoutes.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  {
+    path: '/account/webhooks',
+    redirect: '/webhooks'
+  },
+  {
+    path: '/reset-password',
+    public: true,
+    forceLogout: true,
+    component: passwordReset
+  }
```
Comment (bkemper) [inline src/config/emailRoutes.js]: Do we need this extra file or should we just include this in the `config/routes.js`.

File: src/config/emailRoutes.js (hunk 1 above)
Comment (jonambas) [reply src/config/emailRoutes.js]: I think ProtectedRoute should handle the redirect to the correct route

File: src/config/emailRoutes.js (hunk 3 above)
Comment (jonambas) [reply src/config/emailRoutes.js]: Wanted to keep these separate to easily remove later

**Pull request merged**